packages/vaadin-virtualizer/src/async.js

```diff
@@ -0,0 +1,208 @@
+/**
```

Comment:
This file is a copy of https://github.com/Polymer/polymer/blob/master/lib/utils/async.js